commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/src/main/resources/com/liferay/commerce/theme/speedwell/site/initializer/internal/dependencies/journal_articles/SPEEDWELL_ACCOUNT.xml

```diff
@@ -1,61 +1,61 @@
 <?xml version="1.0"?>
 
 <root available-locales="en_US" default-locale="en_US">
-	<dynamic-element index-type="keyword" instance-id="drrp" name="PageTitle" type="text">
-		<dynamic-content language-id="[£LOCALE£]"> <![CDATA[My Account]]> </dynamic-content>
+	<dynamic-element index-type="keyword" instance-id="my-account-title" name="PageTitle" type="text">
```

Comment:
@Commando18 why we are removing the parameter?

Reply:
Because it doesn't make sense to refresh the page if you click the title